.travis.yml

```diff
@@ -12,6 +12,7 @@ matrix:
         - php: 5.6
           env: SYMFONY_VERSION="~3.0@dev" COMPOSER_FLAGS="--prefer-stable"
         - php: 7.0
+          env: LINT_TEST_CASES=1
```

Comment:
I think that for now we could remove that line, merge the branch (so no changes for Travis) and then, when we get all test base valid, we will add it again. What do you think?